docs/nddata/utils.rst

```diff
@@ -0,0 +1,257 @@
+.. _nddata_utils:
+
+astropy.nddata.utils
+====================
+
+Overview
+--------
+
+The `astropy.nddata.utils` module includes general utility functions
+for array operations.
+
+
+Cutout Images
```

Comment:
Can you please link back from the `Cutout` class docstring to this high-level docs section?
Makes it easy to learn how it works and go back and forth in the HTML docs.